conan/cli/commands/compare.py

```diff
@@ -0,0 +1,233 @@
+import json
+import subprocess
+import os
+
+from conan.errors import ConanException
+from conan.api.conan_api import ConanAPI
+from conan.api.output import ConanOutput
+from conan.cli.command import conan_command
+from conan.internal.util.files import chdir
+from conan.api.model import RecipeReference, ListPattern
+
+def output_json(msg):
+    return json.dumps({"greet": msg})
+
+
+@conan_command(group="Security", formatters={"json": output_json})
```

Comment:
For this to be consistent with the rest of the commands this should have a plain text output and a html formatter, instead of writing to a  file directly... I don't know if that's what we want to do or not for this command...

Reply:
It is possible to have a command that doesn't fully align with the formatters, and it always write a file to disk, like the ``conan lock create`` command, which doesn't generate the file via formatters.